codecademy/day_at_the_supermarket.py

```diff
@@ -29,3 +29,44 @@ def compute_bill(food):
             print(str(item) + " out of stock")
     print("total: " + str(total))
     return total
+
+
+#  4 Lists + Functions
+"""The function counts str 'fizz' in the input list"""
+
+
+def fizz_count(x):
+    count = 0
+    for item in x:
+        if item == 'fizz':
+            count += 1
+    return count
+
+
+print(fizz_count(['fizz', 'blits', 'fizz']))
+
+#  11 Making a Purchase
+"""Calculate the sum of prices for fruit's list"""
```

Comment:
is this a docstring for the `compute_bill_11()` function?
If yes then it must be on next line right after the function definition line. see https://www.python.org/dev/peps/pep-0257/#one-line-docstrings